src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImageInteractor.cpp

```diff
@@ -514,11 +514,31 @@ void medVtkViewItkDataImageInteractor::setWindowLevelFromMinMax()
 
     if(sender)
     {
-        double minIntensity = d->minIntensityParameter->value();
-        double maxIntensity = d->maxIntensityParameter->value();
+        double minIntensityValue = d->minIntensityParameter->value();
+        double maxIntensityValue = d->maxIntensityParameter->value();
 
-        double level = 0.5 * (maxIntensity - minIntensity) + minIntensity;
-        double window = maxIntensity - minIntensity;
+        if( sender == d->minIntensityParameter && minIntensityValue >= maxIntensityValue )
```

Comment:
Can you explain yours modifications

Reply:
there are no modifications, only indentation from line 509 in red, and i added comments lines 522 and 530 in green :).
Github is not really good with indentation

Reply:
ok